workspaces/homepage/plugins/dynamic-home-page/src/components/EntitySection/EntityCard.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+
+import { Link } from '@backstage/core-components';
+
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+
+import TagList from './TagList';
+
+interface EntityCardProps {
+  link: string;
+  title: string;
+  version?: string;
+  description: string;
+  tags: string[];
+  kind: string;
+}
+
+const EntityCard: React.FC<EntityCardProps> = ({
+  link,
+  title,
+  description,
+  tags,
+  kind,
+}) => {
+  return (
+    <Box
+      sx={{
+        border: theme => `1px solid ${theme.palette.grey[400]}`,
+        borderRadius: 3,
+        overflow: 'hidden',
+      }}
+    >
```

Comment:
Please use a Card instead. You can remove the custom borderRadius since our latest theme added this bigger border radius already. If a customer uses their own theme, there is no reason that the home page cards enforces that.

Can we also make the content scrollable instead of saying "overflow hidden" when the card is too small?